.github/workflows/static.yml

```diff
@@ -10,19 +10,20 @@ jobs:
   build:
     name: static
     runs-on: ubuntu-latest
-    strategy:
-      matrix:
-        python-version: ["3.8", "3.11"]
     steps:
       - uses: actions/checkout@v3
-      - name: Set up Python ${{ matrix.python-version }}
+
+      - name: Set up Python
         uses: actions/setup-python@v3
         with:
-          python-version: ${{ matrix.python-version }}
-      - name: Install dependencies
+          python-version: "3.10"
+
+      - name: Install python dependencies
         run: |
-          python -m pip install --upgrade pip
-          python -m pip install -r requirements-${{ matrix.python-version }}.txt
-          python -m pip install . # Install the elfpy module
+          python -m pip install stochastic --ignore-requires-python
+          python -m pip install -r requirements.txt
+          ape plugins install .
+          python -m pip install .
+
       - name: Analysing code with pyright
         run: python -m pyright $(git ls-files '*.py' ':!:*apeworx*')
```

Comment:
still ignoring apeworx here, but not in pylint

Reply:
passes in 192cb98cdca500be3f69865ab4ad4926962f78ac